DatadogSessionReplay/Sources/Processor/SnapshotProcessor.swift

```diff
@@ -96,8 +96,7 @@ internal class SnapshotProcessor: SnapshotProcessing {
         var records: [SRRecord] = []
         // Create records for describing UI:
         if viewTreeSnapshot.context.applicationID != lastSnapshot?.context.applicationID ||
-            viewTreeSnapshot.context.sessionID != lastSnapshot?.context.sessionID ||
-            viewTreeSnapshot.context.viewID != lastSnapshot?.context.viewID {
```

Comment:
/question Is this also part of the fix?

Reply:
Reverted. Will log a separate ticket for the double full snapshot issue.